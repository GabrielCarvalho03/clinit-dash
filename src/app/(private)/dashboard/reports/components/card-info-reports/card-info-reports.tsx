import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";

export type CardInfoReportsProps = {
  title: string;
  text: string;
};

export const CardInfoReports = ({ text, title }: CardInfoReportsProps) => {
  return (
    <Card className="flex-1">
      <CardHeader className="pb-2">
        <CardTitle className="text-sm font-medium">{title}</CardTitle>
      </CardHeader>
      <CardContent>
        <p className="text-2xl font-bold">{text}</p>
      </CardContent>
    </Card>
  );
};
