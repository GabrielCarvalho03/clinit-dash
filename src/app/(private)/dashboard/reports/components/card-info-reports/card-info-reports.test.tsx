import { render } from "@testing-library/react";
import { CardInfoReports } from "./card-info-reports";

describe("CardInfoReports", () => {
  it("should render CardInfoReports correctly", () => {
    const { asFragment } = render(<CardInfoReports title="Test" text="Test" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
