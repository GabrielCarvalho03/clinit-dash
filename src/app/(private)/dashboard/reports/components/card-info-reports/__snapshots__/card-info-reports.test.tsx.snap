// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`CardInfoReports should render CardInfoReports correctly 1`] = `
<DocumentFragment>
  <div
    class="bg-card text-card-foreground flex flex-col gap-6 rounded-xl border py-6 shadow-sm flex-1"
    data-slot="card"
  >
    <div
      class="@container/card-header grid auto-rows-min grid-rows-[auto_auto] items-start gap-1.5 px-6 has-data-[slot=card-action]:grid-cols-[1fr_auto] [.border-b]:pb-6 pb-2"
      data-slot="card-header"
    >
      <div
        class="text-sm font-medium"
        data-slot="card-title"
      >
        Test
      </div>
    </div>
    <div
      class="px-6"
      data-slot="card-content"
    >
      <p
        class="text-2xl font-bold"
      >
        Test
      </p>
    </div>
  </div>
</DocumentFragment>
`;
